test/lib/jdk/test/lib/Asserts.java

```diff
@@ -607,6 +612,39 @@ public static void assertStringsEqual(String str1, String str2,
         }
     }
 
+    /**
+     * Asserts that contents of two files are equal.
+     *
+     * @param f1 The path of the first file to compare
+     * @param f2 The path of the second file to compare
+     * @throws RuntimeException on mismatch
+     * @throws IOException on I/O failures
+     */
+    public static void assertFileContentsEqual(Path f1, Path f2) throws IOException {
```

Comment:
Hello Volkan, is this new method needed? Can its call sites instead be replaced with `java.nio.file.Files.mismatch(...)` call?

Reply:
I thought it reports more useful diagnostics compared to the earlier `file compare failed` message. Nevertheless, replaced it with `Files::mismatch` in 421d19d468d62bdb04aee458c72a338a5c053e73.